jdbc-adapter/integration-test-data/run_integration_tests.sh

```diff
@@ -81,45 +81,40 @@ cleanup_remote_dbs() {
 	$docker_helper --rm $config
 }
 
-prepare_configuration_dir() {
-	mkdir -p "$1"
-	cp integration-test-data/EXAConf "$1/EXAConf"
-}
-
-prepare_data_dir() {
-	mkdir -p "$1"
-	dd if=/dev/zero of="$1/dev.1.data" bs=1 count=1 seek=4G
-	touch "$1/dev.1.meta"
+prepare_docker() {
+	docker pull "$docker_image"
+	git clone https://github.com/EXASOL/docker-db.git $1/$docker_name
```

Comment:
"use double quotes."
Parameterize base URL to allow testing with local repository.

Reply:
done